README.md

```diff
@@ -1 +1,39 @@
-# Newsweek-Clone
\ No newline at end of file
+# Newsweek Clone
+
+Project Live Version: [https://www.newsweek.com/]
```

Comment:
- [ ] This should be the link to your deployed project version. Please deploy the project using github pages and include the link in this section of the readme.